README.rst

```diff
@@ -1,3 +1,18 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/l/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/wheel/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/pyversions/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://travis-ci.org/Pylons/waitress.svg?branch=master
+    :target: https://travis-ci.org/Pylons/waitress
+
```

Comment:
I'm confused. What gets released to PyPI? Are there multiple "branches" on PyPI? What do you mean by "when new branches are released"?

If you want to have separate badges depending on which Git branch the code is in, then whenever you do a branch, you'd have to remember to modify README.rst manually. I've never seen any other project do this. It seems everyone understands that the badges are either for the master branch, or for the release on PyPI (or NPM or whatever).

I'll add `:alt:` attributes to the images.